recipes/lightfm/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "lightfm" %}
+{% set version = "1.14" %}
+{% set sha256 = "4646742192f114cc07964cf463a370af6786f4027e46d32fff45fbf7fbce3d7f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+    - cython
+    - gcc  # [linux]
+    # Pin numpy at build time with the minimum version available.
+    - numpy 1.10.*  # [not win]
+    - numpy 1.11.*  # [win]
+    - scipy >=0.17
+    - requests
+  run:
+    - python
+    - libgcc  # [linux]
+    - numpy >=1.10  # [not win]
+    - numpy >=1.11  # [win]
+    - scipy >=0.17
+    - requests
+
+test:
+  imports:
+    # ensure OpenMP backend is used on linux
+    - lightfm._lightfm_fast_openmp  # [linux]
+    - lightfm
+    - lightfm.lightfm
+
+about:
+  home: https://github.com/lyst/lightfm
+  license: Apache-2.0
+  license_family: Apache
+  # license file not included in the .tar.gz, should be fixed in the next release.
+  # license_file: LICENSE.txt
```

Comment:
Can you please manually package the license? https://conda-forge.org/docs/meta.html#packaging-the-license-manually

Reply:
Yes, done. Thanks for catching that.